package.json

```diff
@@ -103,24 +123,8 @@
     "semantic-release": "^17.4.7",
     "typescript": "3.9.6"
   },
-  "files": [
-    "bin",
-    "dist"
-  ],
-  "nyc": {
-    "all": true,
-    "include": [
-      "dist/htmlhint.js"
-    ],
-    "exclude": [
-      "**/*.spec.js",
-      "src/**/*.ts",
-      "bin/htmlhint"
-    ],
-    "reporter": [
-      "text",
-      "lcov"
-    ],
-    "sourceMap": false
+  "engines": {
+    "node": ">=16.13.0",
```

Comment:
I'd probably drop both of these unless there is something needed in order for this to run once installed. I think there is a minimum version for stuff like ESM, but I think it's 12.20 something

Reply:
Yeah, actually, you're right, after thinking about it, probably better to remove this 🙏